frontend/ui/src/pages/Inbox/MessageInput/InputOptions.tsx

```diff
@@ -97,6 +133,27 @@ export const InputOptions = ({source, inputDisabled, input, setInput, selectTemp
           <TemplateAlt aria-hidden className={styles.templateAltIcon} />
         </div>
       </button>
+
+      {maxFileSizeErrorPopUp && (
+        <div className={styles.fileSizeErrorPopUp}>
+          <ErrorPopUp
+            message="Failed to upload the file. The maximum file size allowed is 25MB."
+            closeHandler={closeFileErrorPopUp}
+          />
+        </div>
+      )}
+
+      {mediaComponentConfig.enabled && source === ('facebook' || 'instagram') && (
+        <button className={`${styles.iconButton}`} type="button" disabled={inputDisabled}>
+          <div className={styles.actionToolTip}>Files</div>
+
+          <label htmlFor="file" className={styles.filesLabel}>
+            <Paperclip aria-hidden className={styles.paperclipIcon} />
+          </label>
+
+          <input type="file" id="file" name="file" onChange={selectFile} className={styles.fileInput} />
+        </button>
+      )}
     </div>
```

Comment:
so normally we just want to disable the upload button if the media-resolver component in the` /client.config` is not running: here I'm also disabling it if the source is not instagram or facebook because we haven't added support for sending attachments for others source yet. (but obviously this has to be changed as soon as we add support for others sources)